tenant-isolation-config-service-api/src/main/proto/buf.yaml

```diff
@@ -0,0 +1,8 @@
+version: v1
```

Comment:
make sure to declare this new source root in the top level work yaml.

Reply:
done